src/libslic3r/PrintObject.cpp

```diff
@@ -3306,6 +3306,13 @@ void PrintObject::discover_horizontal_shells()
 // fill_surfaces but we only turn them into VOID surfaces, thus preserving the boundaries.
 void PrintObject::combine_infill()
 {
+    auto max_layer_height = [this](int idx_nozzle) {
+        auto    &print_config = this->print()->config();
+        coordf_t mh           = print_config.max_layer_height.get_at(idx_nozzle - 1);
```

Comment:
should we use `idx_nozzle` instead here?

Reply:
Ah that's right.